Jenkinsfile

```diff
@@ -86,6 +86,41 @@ try {
                         stage('Build') {
                             sh "${params}"
                         }
+                        
+                        set_gh_status_pending(is_pr, 'Testing with QEMU')
+                        params = ["${script_env_global}", "${script_env_local}",
+                                  "docker/test-qemu.sh"].join("\n")
+                        stage("Test QEMU") {
```

Comment:
I think testing should run after "store images" stage, because testing can change state of build results directory